package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "example-storefront",
-  "version": "2.9.0",
+  "version": "3.0.0",
```

Comment:
Is this OK as just ` 3.0.0`, or should it be `3.0.0-alpha`?

Reply:
Changed to the latter.